tests/jobs/correlated_observations_scaling/unit/test_scaling_config.py

```diff
@@ -57,12 +55,10 @@ def test_valid_job(
         "CALCULATE_KEYS": {"std_cutoff": 1.0e-6, "alpha": 3},
         "UPDATE_KEYS": {},
     }
-    reporter = FileReporter(os.path.realpath(os.getcwd()))
-    job = ScalingJob(
-        obs_keys, [], obs_with_data, user_config_dict, reporter, default_vals
-    )
-    assert job.get_index_lists() == scaling_job_content["get_index_lists"]
-    assert job.get_calc_keys() == scaling_job_content["get_calc_keys"]
+
+    config = ObsCorrConfig(user_config_dict, obs_keys, default_vals)
```

Comment:
`test_valid_configuration` I guess. This just makes more sense as well, not having to initiate a `FileReporter` and the entire `ScalingJob` when we're only asserting parts of the configuration